lib/ansible/modules/source_control/gitlab_group.py

```diff
@@ -110,6 +116,7 @@
 
 from ansible.module_utils.basic import *
 from ansible.module_utils.pycompat24 import get_exception
+from ansible.modules.extras.source_control.gitlab_user import GitLabUser
```

Comment:
I think your PR is against best practices (http://docs.ansible.com/ansible/dev_guide/developing_modules_best_practices.html#common-pitfalls)
_"Modules should be the building blocks. Instead of creating a module that does the work of other modules, use Plays and Roles instead."_